sys/linux/socket.txt

```diff
@@ -317,6 +321,41 @@ ifr_ifru [
 
 define IFREQ_SIZE	sizeof(struct ifreq)
 
+# https://elixir.bootlin.com/linux/v5.3.11/source/include/uapi/linux/if_vlan.h
+vlan_args {
+	cmd		flags[vlan_cmd, intptr]
+	device1		array[int8, 24]
```

Comment:
Looking at the code this seems to be a device name:
https://elixir.bootlin.com/linux/latest/source/net/8021q/vlan.c#L565
if so it's better to do:
```
device1 string[devnames, 24]
```
Or syzkaller will have hard time guessing 24 random bytes that happen to represent a valid device name :)

Reply:
Wasn't aware that there is devnames available, updated.